Mathlib/FieldTheory/Differential/Basic.lean

```diff
@@ -0,0 +1,146 @@
+/-
+Copyright (c) 2024 Daniel Weber. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Daniel Weber
+-/
+import Mathlib.Algebra.Algebra.Field
+import Mathlib.Algebra.EuclideanDomain.Field
+import Mathlib.Data.Countable.Small
+import Mathlib.LinearAlgebra.Basis.VectorSpace
+import Mathlib.RingTheory.Derivation.DifferentialRing
+import Mathlib.RingTheory.LocalRing.Basic
+import Mathlib.Tactic.FieldSimp
+
+/-!
+# Differential Fields
+
+This file defines the logarithmic derivative `logd` and Liouville field extensions,
+as a preparation for Liouville's theorem.
+-/
+
+open Differential algebraMap
+
+variable {R : Type*} [Field R] [Differential R] (a b : R)
+
+/--
+The logarithmic derivative of a is a′ / a.
+-/
+def logd : R := a′ / a
```

Comment:
logDeriv is already used in `Mathlib.Analysis.Calculus.LogDeriv`, so I'm changing it back to `logd`

Reply:
Oh, but that's good because they are morally the same object. What it means instead is that your API should be namespaced (under `Derivation` probably?)

Reply:
I think `Differential` is a better namespace

Reply:
`Differential` might be a bit unclear given that it also applies to the other `logDeriv`